tests/utils.py

```diff
@@ -601,15 +601,23 @@ def wait_pay():
         # wait for sendpay to comply
         self.rpc.waitsendpay(rhash)
 
-    def fake_bitcoind_fail(self, exitcode):
+    def fake_bitcoind_fail(self, failscript='exit 1', cmd=None):
         # Create and rename, for atomicity.
         f = os.path.join(self.daemon.lightning_dir, "bitcoin-cli-fail.tmp")
         with open(f, "w") as text_file:
-            text_file.write("%d" % exitcode)
-        os.rename(f, os.path.join(self.daemon.lightning_dir, "bitcoin-cli-fail"))
+            text_file.write(failscript)
```

Comment:
We'll definitely need to change this, a python script writing a python script that is then run by a subprocess, that's an antipattern if I ever saw one :smile: